src/BlazorUI/Bit.BlazorUI/Components/DatePicker/BitDatePicker.razor.cs

```diff
@@ -151,6 +152,13 @@ public CultureInfo Culture
     /// </summary>
     [Parameter] public bool ShowWeekNumbers { get; set; }
 
+    /// <summary>
+    /// The tabIndex of the TextField
+    /// </summary>
+    [Parameter] public int TabIndex { get; set; }
+
+    public string ActiveDescendantId { get; set; } = Guid.NewGuid().ToString();
+    public string CalloutId { get; set; } = string.Empty;
```

Comment:
should these properties be public? or even be property? are these part of the public API of the component?